airflow/providers/amazon/aws/example_dags/example_redshift_sql.py

```diff
@@ -15,24 +15,20 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-"""
-This is an example dag for using `RedshiftSQLOperator` to authenticate with Amazon Redshift
-then execute a simple select statement
-"""
```

Comment:
Why remove this?

Reply:
I dont think it's necessary since all files in this directory are example dags. It's kind of obvious. Plus, most of sample dags in this directory do not have a top level comment like this one